testing/src/main/java/io/grpc/testing/TestUtils.java

```diff
@@ -151,11 +153,7 @@ public static ServerInterceptor delayServerResponseInterceptor(final AtomicLong
           @Override
           public void sendPayload(RespT payload) {
             if (delayMillis.get() != 0) {
-              try {
-                Thread.sleep(delayMillis.get());
-              } catch (InterruptedException e) {
-                Throwables.propagate(e);
-              }
+              Uninterruptibles.sleepUninterruptibly(delayMillis.get(), TimeUnit.MILLISECONDS);
```

Comment:
Why this change? If the thread was interrupted, I would think we would want to throw.